@here/harp-mapview/test/ImageCacheTest.ts

```diff
@@ -246,8 +246,8 @@ describe("MapViewImageCache", function() {
     it("#remove image 2", function() {
         const cache = new MapViewImageCache();
 
-        cache.addImage("testImage1", "httpx://naxos.de");
-        cache.addImage("testImage2", "httpx://naxos.de");
+        cache.addImage("testImage1", "httpx://naxos.de", false);
+        cache.addImage("testImage2", "httpx://naxos.de", false);
```

Comment:
Last flag to false avoids the actual url fetch cause the promise rejection. The fetch is not needed for these particular tests.